### ISSUE TITLE
Work with vim-fugitive

### ISSUE THREAD

**Issue by jrock2004** [rhysd/committia.vim#28]
So are you saying that I can use this with vim-fugitive? I have tried setting `let g:committia_open_only_vim_starting = 1` but when I do `:Gcommit` it brings up the default committing window

![image](https://cloud.githubusercontent.com/assets/655716/20038758/13657bfa-a407-11e6-95bf-acf465cac7af.png)

**Comment by rhysd** [comment]
> So are you saying that I can use this with vim-fugitive?

No. vim-fugitive opens commit buffer with uncommon way. So currently commitia.vim can't handle it correctly.
When `let g:committia_open_only_vim_starting = 1`, committia.vim do nothing on `:Gcommit`. Then you can avoid the collision between committia.vim and vim-fugitive.

**Comment by macintacos** [comment]
Just a note, I think the same thing happens with [neogit](https://github.com/TimUntersberger/neogit).

**Comment by towry** [comment]
This works, https://github.com/yuki-yano/dotfiles/blob/0a1bb68e0e9bc5332d366fcf5aec71d1c1468598/.vim/lua/plugins/git.lua#L80

Need to manually open it.
